environment.yml

```diff
@@ -50,12 +50,11 @@ dependencies:
 - urllib3=1.25.7
 - xgboost=0.81
 - pip:
-  - civis==1.15.1
+  - git+https://github.com/civisanalytics/civis-python.git@civis-1835-v1.16.0-release#egg=civis
   - civisml-extensions==0.2.1
   - dropbox==9.4.0
   - ftputil==3.4
   - muffnn==2.3.1
-  - pubnub==4.3.0
```

Comment:
I'm dropping the `pubnub` requirement here at datascience-python, but I don't think we should count this as a breaking change for datascience-python. PubNub was really a dependency of civis-python, and I've completely removed all code related to PubNub from civis-python (https://github.com/civisanalytics/civis-python/pull/425). I don't think any datascience-python users would really call the `pubnub` package directly.